lib/Onyx.js

```diff
@@ -918,11 +941,12 @@ function multiSet(data) {
  * Merges an array of changes with an existing value
  *
  * @private
- * @param {Array<*>} changes Array of changes that should be applied to the existing value
  * @param {*} existingValue
+ * @param {Array<*>} changes Array of changes that should be applied to the existing value
+ * @param {bool} omitNull Whether null top-level null values should be omitted
  * @returns {*}
  */
-function applyMerge(changes, existingValue) {
+function applyMerge(existingValue, changes, omitNull = true) {
```

Comment:
This is a tricky one. If we are using this method to build the "batch of changes" then don't we actually want to _drop_ those keys from storage and not just set them to "null" or is it the same thing as far as SQLite is concerned?

The only reason to "omit" them from the final object is so they get removed from storage.

Reply:
Maybe it would be clearer to explicitly remove the top level null keys with some brief explanation _after_ we get the "result" (i.e. `modifiedData`) and basically make this method not responsible for modifying the object in that way?